tests/common/test_oresat_file_cache.py

```diff
@@ -0,0 +1,299 @@
+import sys
+sys.path.append('../../')
+
+import unittest
+
+import threading
+
+from time import sleep
+
+from shutil import rmtree
+
+from os import rmdir, remove, listdir
+
+from os.path import basename, abspath, exists
+
+from olaf.common.oresat_file_cache import OreSatFileCache
+
+from olaf.common.oresat_file import new_oresat_file
```

Comment:
Don't need spaces between all imports.

Import style used in OLAF is:
python core imports group
other library imports group
internal (OLAF) imports group

Should have a new line between each group of imports.